nexus/tests/integration_tests/endpoints.rs

```diff
@@ -416,6 +416,14 @@ lazy_static! {
             group_attribute_name: None,
         };
 
+    pub static ref DEMO_SYSTEM_METRICS_URL: String =
+        format!(
+            "/system/metrics/virtual_disk_space_provisioned?start_time={:?}&end_time={:?}&id={}",
```

Comment:
Not something we need to deal with right now, but it seems like we'll want some other kind of coverage test for this particular endpoint, since giving different metric names or query parameters would result in different authz checks.

Reply:
Sure; I refactored the test a bit to make it more obvious there's currently only one auth check happening for the whole group of metrics that are currently provided.